templates/src/middleware/auth/jwtDecoder.js

```diff
@@ -0,0 +1,20 @@
+const expressJWT = require("express-jwt");
+const pathToRegexp = require('path-to-regexp');
+const { jwtSecret } = require("../../conf");
+
+/*the middleware to verify token and parse token to req.user, 
+please visit https://github.com/auth0/express-jwt for the more details */
+const jwtDecoder = expressJWT({
```

Comment:
Why is there JWT code here? The service will never receive a JWT, as it's handled by the access proxy.

Reply:
I thought access proxy may pass the access token to the endpoints instead of the plain header.  Otherwise, the endpoints are directly exposed without security protection if the access bypasses the proxy.

Reply:
restored the authentication component and added the whitelist filter.